### ISSUE TITLE
Using coroutines with multiple databases

### ISSUE THREAD

**Issue by anthony-tarr** [JetBrains/Exposed#624]
I'm attempting to do something like the following:

```
newSuspendedTransaction(Dispatchers.IO, db1) {
  val user = repo1.getUserFromUUID(uuid)

  ...

  newSuspendedTransaction(Dispatchers.IO, db2) {
    val newUser = repo2.createUser(user.email)
    repo2.createPassword(newUser, password)
  }

  ...
}
```

The reason why both are `newSuspendedTransaction` is because I'm abstracting the transaction out of the repo. But I have no way to manually reach into the existing `Transaction` or fetch the `TransactionScope` directly.

But the second transaction doesn't execute in the correct database. I've verified it's passed in correctly.

Am I using the coroutine helpers correctly, or is there another workaround?

**Comment by Tapac** [comment]
fixed in 0.16.4, please try it.

**Comment by anthony-tarr** [comment]
Works now, thanks!
